packages/node-sdk/paima-db/migrations/up.sql

```diff
@@ -241,3 +241,20 @@ CREATE TABLE cde_cardano_mint_burn(
   output_addresses JSONB NOT NULL,
   PRIMARY KEY (cde_id, tx_id)
 );
+
+CREATE TABLE achievement_type(
+  name TEXT NOT NULL PRIMARY KEY,
+  is_active BOOLEAN NOT NULL DEFAULT true,
+  display_name TEXT NOT NULL,
+  description TEXT NOT NULL DEFAULT '',
```

Comment:
Did we want to include these two in the db as-is despite the fact they're language-specific depending on the header field you use when querying the achievement endpoint?

Reply:
If a table for language-agnostic info + another one for per-language info works for you, we can do that

Reply:
In an ideal world translations wouldn't be in the database at all, but if it has to be, probably it's best for it to be in a separate table

Reply:
Possible alternatives:
1. Another export from `packaged/endpoints.cjs`
2. Another `.json` file in `packaged/`

Reply:
Since I find putting "code" (stuff that's defined by the programmer and not by user input) in the database kind of goofy, this is what I'm looking at now:

* Game ID, achievement types, and languages are an additional `packaged/endpoints.cjs` export
  * But it's just a big data object, not a class with methods on it
* Achievement progress is stored in a DB table